experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/unboxed.spec.ts

```diff
@@ -211,6 +279,254 @@ describe("unboxed unit tests", () => {
 				assert.equal(unboxed.value, true);
 			});
 		});
+
+		describe("Value field", () => {
```

Comment:
The difference in implementation between unboxedField for value and optional fields in one check for empty thats only done for optional nodes. Even ignoring that, tne entore logic for handleing value fields in there is a couple short lines. THis is hundrends of likes of tests for a couple short lines of implementation. Thats way too much. They also have a lot of dependencies beyond whats being tested (tree setup, unrelated unboxing functions, nodes, etc), which should make us error on the side of having fewer tests.